playground/frontend/lib/modules/editor/components/share_dropdown/share_tabs_headers.dart

```diff
@@ -35,17 +35,19 @@ class ShareTabsHeaders extends StatelessWidget {
   Widget build(BuildContext context) {
     final appLocale = AppLocalizations.of(context)!;
 
-    return Consumer<PlaygroundState>(builder: (context, state, child) {
-      return SizedBox(
-        width: _width,
-        child: TabBar(
-          controller: tabController,
-          tabs: [
-            Text(appLocale.link),
-            Text(appLocale.embed),
-          ],
-        ),
-      );
-    });
+    return Consumer<PlaygroundController>(
+      builder: (context, controller, child) {
+        return SizedBox(
+          width: _width,
```

Comment:
The name could be more specific: `_tabBarWidth`.

Reply:
This was only indented but not added. I would rather not add more refactoring.

This PR focuses on establishing a more or less stable interface between the two apps and the shared package. All the internals of the three will be addressed later.